starlark/testdata/builtins.star

```diff
@@ -186,6 +186,9 @@ assert.eq(str(getattr(myset, "union")), "<built-in method union of set value>")
 assert.fails(lambda: getattr(myset, "onion"), "no .onion field or method")
 assert.eq(getattr(myset, "onion", 42), 42)
 
+assert.eq(sorted(dir("")), dir("")) # dir returns a sorted list
+dir("").append("!") # dir returns a mutable list
```

Comment:
Test that subsequent calls to dir don't return the mutated list?

Reply:
Done.